introduction.md

```diff
@@ -1 +1,2 @@
-i am new dojo_urles master, obey me slaves. yes and btw good ruby path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?